### PR TITLE
changed styling of polls in sections and articles and home page

### PR DESCRIPTION
**Pull request by codiebeulaine**
@peterp ready for review

### CODE REVIEW

**Comment by codiebeulaine** [comment]
screen shot 2016-04-15 at 10 02 45 am

File: freebasics/templates/polls/poll_page_in_section.html [hunk 1]
```diff
@@ -2,15 +2,22 @@
 
 {% block content %}
 	{% if questions %}
+	<div class="block-poll fb-body fb-card">
+		<ul>
 		{% for question in questions %}
-
-				<h3>{{ question.title }}</h3>
+			{% if forloop.first%}
+			<h6 class="block-heading fb-accent-1">{% trans "What's your opinion?" %}</h6>
```
Comment (peterp) [inline freebasics/templates/polls/poll_page_in_section.html]: Shouldn't this be outside of the `ul` and loop?

Comment (peterp) [inline freebasics/templates/polls/poll_page_in_section.html]: This should be moved outside the forloop and `ul` element.

File: freebasics/templates/polls/poll_page.html [hunk 2]
```diff
@@ -3,20 +3,21 @@
 {% block content %}
 	{% if questions %}
 	<div class="block-poll fb-body fb-card">
+		<ul>
 		{% for question in questions %}
 			{% if forloop.first%}
```
Comment (peterp) [inline freebasics/templates/polls/poll_page.html:9]: Same issue here, this should move outside of the `ul`

**Comment by peterp** [comment]
:+1:, once the `<ul><h6>` issue is resolved.

File: freebasics/templates/polls/poll_page.html (hunk 2 above)
Comment (codiebeulaine) [inline freebasics/templates/polls/poll_page.html:9]: @peterp the ul styling is found in the block-poll class though?

**Pull request merged**